cmake/find/Findodb-compiler.cmake

```diff
@@ -0,0 +1,366 @@
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# Instructions:
+# 
+# odb_compile command takes a target as first argument, so *AFTER* either:
+#   add_executable("target" ...)
+#   add_library("target" ...)
+#
+# odb_compile just forwards options to the command line of the compiler, 
+# more information: http://www.codesynthesis.com/products/odb/doc/odb.xhtml
+#
+# the call options are:
+#   odb_compile("target" 
+#       FILES                     # header files to be parsed *REQUIRED*
+#         persistent_object1.hxx
+#         persistent_object2.hxx
+#       DB                        # --database *REQUIRED*
+#         sqlite
+#       INCLUDE                   # -I dir
+#         /usr/include/dir
+#         /home/example/include
+#       GENERATE_QUERY            # --generate-query
+#       GENERATE_SESSION          # --generate-session
+#       GENERATE_SCHEMA           # --generate-schema
+#       SCHEMA_FORMAT             # --schema-format
+#         embedded
+#       SCHEMA_NAME               # --schema-name
+#         schema_name
+#       TABLE_PREFIX              # --table-prefix
+#         prefix
+#       STANDARD                  # --std 
+#         c++11
+#       SLOC_LIMIT                # --sloc-limit
+#         99999
+#       ODB_PROLOGUE              # --odb-prologue
+#         "\#include \"this\""
+#       HEADER_PROLOGUE           # --hxx-prologue
+#         "header prologue"
+#       INLINE_PROLOGUE           # --ixx-prologue
+#         "inline prologue"
+#       SOURCE_PROLOGUE           # --cxx-prologue
+#         "source prologue"
+#       ODB_EPILOGUE              # --odb-epilogue
+#         "odb epilogue"
+#       HEADER_EPILOGUE           # --hxx-epilogue
+#         "header epilogue"
+#       INLINE_EPILOGUE           # --ixx-epilogue
+#         "inline epilogue"
+#       SOURCE_EPILOGUE           # --cxx-epilogue
+#         "header epilogue"
+#       ODB_PROLOGUE-FILE         # --odb-prologue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       HEADER_PROLOGUE-FILE      # --hxx-prologue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       INLINE_PROLOGUE-FILE      # --ixx-prologue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       SOURCE_PROLOGUE-FILE      # --cxx-prologue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       ODB_EPILOGUE-FILE         # --odb-epilogue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       HEADER_EPILOGUE-FILE      # --hxx-epilogue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       INLINE_EPILOGUE-FILE      # --ixx-epilogue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#       SOURCE_EPILOGUE-FILE      # --cxx-epilogue-file
+#         ${CMAKE_SOURCE_DIR}/file.ext
+#    )
+# ----------------------------------------------------------------------
+
+if(NOT HUNTER_ENABLED)
+  macro(hunter_status_debug)
+    message(${ARGV})
+  endmacro()
+endif()
+
+if(ANDROID) 
+  #AWP: this seems to be only needed by the Android toolchain, maybe this should
+  #     be elsewhere?
```

Comment:
Ok, I will look into the `install` and how can it be applied here.

FYI: this additional code is also needed by Findodb.cmake to find the libraries inside hunter's structure for packages. 
I am not sure if this is related but the Android toolchain has this line: https://github.com/ruslo/polly/blob/master/os/android.toolchain.cmake#L1532 which was the motivation for me adding this extra code. After adding this extra code, it managed to find the library and the compiler. I tested without the extra code and they were not being found.